spec/models/setting_spec.rb

```diff
@@ -54,6 +54,7 @@
       expect(described_class.alert_via_sentry?).to be true
       expect(described_class.means_test_review_phase_one?).to be false
       expect(described_class.partner_means_assessment?).to be false
+      expect(described_class.opponent_organisations?).to be false
```

Comment:
do we also need tests above - for example, wherever `partner_means_assessment` is tested we should have equivalent test for this flag I think

Reply:
I would follow along with changes from this PR to make sure everything is covered in terms of specs 😄 
https://github.com/ministryofjustice/laa-apply-for-legal-aid/pull/5044/files

Reply:
Thank you, Joel. I have added these tests now.